sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/ContinuousExecution.scala

```diff
@@ -334,7 +334,7 @@ class ContinuousExecution(
           epochUpdateThread.interrupt()
           epochUpdateThread.join()
           // The following line must be the last line because it may fail if SparkContext is stopped
-          sparkSession.sparkContext.cancelJobGroup(runId.toString)
+          sparkSession.sparkContext.cancelJobGroup(runId.toString, "continuous execution finished")
```

Comment:
Is it possible to include something similar to a query id within the reason?

Reply:
Modified the reason to include `runId`.